### DISCUSSION TITLE
Prisma Studio Data not reaching main branch

### DISCUSSION THREAD

**Discussion by nichprice** [planetscale/discussion#399] [Q&A] [answered]
Hi!

So I followed the instructions [here](https://planetscale.com/docs/tutorials/prisma-quickstart) to promote my main branch to production and deploy my initial-setup branch to main.

Looks like my deployment was successful. The schema on my main branch updated and it shows the table I created
BUT, when I query the table using the console (select * from Project;), no data comes back.

Whereas when I query my initial-setup branch with the same command, all the data I input via Prisma Studio comes back.

Any thoughts on what I can do to get my data to my main branch?

**Comment by fooforge** [comment, marked as answer]
Hey @nichprice,

Branches are completely isolated from one another. Each branch is essentially its own Vitess cluster, and we don't keep your data automatically in sync between branches with the exception of a `migrations` table that is [often being used](https://planetscale.com/docs/concepts/branching#automatically-copy-migration-data) by ORMs to track changes.

I would recommend to use an artificial or reduced data set for development, but you can also [restore a backup](https://planetscale.com/docs/concepts/back-up-and-restore) to a new branch and go from there.

All that said, we do offer a feature that we call [Data Branching™](https://planetscale.com/docs/concepts/data-branching) which allows you to keep both your schema and data in sync across branches. It is available on our [Team and Enterprise plans](https://planetscale.com/pricing).
